### PR TITLE
Accessibility improvements on club listing page and leaders corner page

### PR DESCRIPTION
**Pull request by csrs**
This is related to issue #263 since it's an accessibility fix. 

Commit a409f9a:
- Added aria-labelledby to sections of the Leaders Corner and the Club listing pages  
- Made some alt text more specific. 

Commit a9c7d5b:
- Added underline to all links, and removed underline from heading-level or more decorative links (in the header, fooder, shadow boxes).

Commit b6bda7d:
- Added p tag to a div that was missing it (this is for screen readers)

### CODE REVIEW

File: src/components/components/club-teaser-list/club-teaser-list.js [hunk 1]
```diff
@@ -20,17 +20,23 @@ export const ClubTeaser = props => {
...
           <img
             src={image || groupSmallImage}
-            alt={title}
+            alt={`${title} club logo`}
             className="club-teaser__image"
           />
         </div>
         <div className="club-teaser__text">
-          <h2 className="club-teaser__title">{title}</h2>
+          <h2 className="club-teaser__title" id={title.replace(/\s/g, "")}>
```
Comment (alexdor) [inline src/components/components/club-teaser-list/club-teaser-list.js]:
```suggestion
          <h2 className="club-teaser__title" id={(title||"").replace(/\s/g, "")}>
```
Could you safeguard the replace call here?

File: src/components/components/faq-group/faq-group.js [hunk 2]
```diff
@@ -13,9 +13,16 @@ export default props => {
     <div className={classes}>
       <ShadowBox>
         <div className="faq__group-header">
-          <h2 title="title title--x-small">{props.header}</h2>
+          <h2 title="title title--x-small" id={props.header.replace(/\s/g, "")}>
```
Comment (alexdor) [inline src/components/components/faq-group/faq-group.js]: Could you safeguard the replace call here from null header?

File: src/components/components/faq-group/faq-group.js [hunk 3]
```diff
@@ -13,9 +13,16 @@ export default props => {
...
+            {props.header}
+          </h2>
         </div>
-        <ul className="faq__group-content">{props.children}</ul>
+        <ul
+          className="faq__group-content"
+          aria-labelledby={props.header.replace(/\s/g, "")}
```
Comment (alexdor) [inline src/components/components/faq-group/faq-group.js]: Could you safeguard the replace call here from null header?

Comment (csrs) [reply src/components/components/faq-group/faq-group.js]: done

File: src/components/components/faq-group/faq-group.js (hunk 2 above)
Comment (csrs) [reply src/components/components/faq-group/faq-group.js]: see commit c3be0c3

File: src/components/components/club-teaser-list/club-teaser-list.js (hunk 1 above)
Comment (csrs) [reply src/components/components/club-teaser-list/club-teaser-list.js]: see commit c3be0c3

File: src/components/base-styles/base/links/_links.scss [hunk 4]
```diff
@@ -15,7 +15,7 @@
 a,
 %link {
   color: color(link);
-  text-decoration: none;
+  // text-decoration: none;
```
Comment (alexdor) [inline src/components/base-styles/base/links/_links.scss]: Could you delete this instead of commenting it out?

Comment (csrs) [reply src/components/base-styles/base/links/_links.scss]: Done, see 690d11c

**Pull request merged**